keywords/hashtag.md

```diff
@@ -1,12 +1,18 @@
 ## Hashtag
 
-A hashtag, denoted by the symbol “#”, is a metadata tag commonly use in the Internet culture, particularly on social media. [^Merriam-Webester24Hashtag]  It originated as a feature on a post on Twitterin August 2007 from a product designer named Chris Messina. The primary purpose was to provide user with an easy way to categorize, discover and participate in specific topics on social media.[^Hastags.org12Whatis#]
+A hashtag, denoted by the symbol “#”, is a metadata tag commonly use in the Internet culture, particularly on social media. [^Merriam-Webester24Hashtag]  It originated as a feature on a post on Twitterin August 2007 from a product designer named Chris Messina. The primary purpose was to provide user with an easy way to categorize, discover and participate in specific topics on social media.[^Hastags.org12Whatis#] In addition, Tweets containing hashtags can link to a specific timeline of other tweets with the same hashtag. [^WikströmHashtagOnTwitter]
+
+Hashtags have significantly impacted the overall performance of social media content which can help small businesses, influencers, brands and many more to reach a bigger audience. A hashtag can contain multiple word without space, such as #dogLover and may also contain numbers. They can be added to social media posts, bios, comments, and other content. They are utilized across various platform, including Instagram, TikTok, YouTube, Facebook, Twitter and many more.[^SproutSocial23HashtagsWhatAreThey] 
```

Comment:
@cathytham You can use back quotes to make monospace fonts, good for coding. For example: `#dogLover`